File: 100-Days-Series/Day 76/SumOfRow.cpp
#include <iostream>
using namespace std;

int main() {
    int nums[3][3] = {
        {1, 4, 9},
        {11, 4, 3},
        {2, 2, 3}
    };
    int targetRow = 1; // Index of the row to sum (second row)
    int sum = 0;

    for (int i = 0; i < 3; i++) {
        sum += nums[targetRow][i];
    }

    cout << "The sum of the numbers in row " << (targetRow + 1) << " is: " << sum << endl;
    return 0;
}

File: 100-Days-Series/Day 76/SumOfRow.js

const nums = [
    [1, 4, 9],
    [11, 4, 3],
    [2, 2, 3]
];

const targetRow = 1; // Index of the row to sum (second row)
let sum = 0;

// Loop through the elements of the target row
for (const num of nums[targetRow]) {
    sum += num;
}

// Print the result
console.log("The sum of the numbers in row " + (targetRow + 1) + " is: " + sum);
